components/blockquote.js

```diff
@@ -0,0 +1,29 @@
+import { Box } from 'theme-ui'
+
+const Blockquote = ({ children }) => {
+  const firstChar = children.props.children.slice(0, 1)
+  const textIndent = firstChar === '“' ? ['-0.4em'] : [0]
+  return (
+    <Box
+      sx={{
+        bg: 'hinted',
```

Comment:
Is it possible to use `variant: 'styles.blockquote'` here instead of manually specifying the `blockquote` styles?